clients/ts/signalr/tests/HubConnection.test.ts

```diff
@@ -1092,11 +1086,14 @@ class TestConnection implements IConnection {
     public sentData: any[];
     public lastInvocationId: string | null;
 
-    constructor() {
+    private sendHandshake: boolean | null;
+
+    constructor(sendHandshake: boolean | null = true) {
```

Comment:
nit: we have the same logic in the .NET tests and we call this parameter `autoHandshake`. Also, I believe we don't want to permit `null` here either, since we have a default value.